utils/config_generator.py

```diff
@@ -77,6 +77,43 @@
     "tr", "uk", "umb", "ur", "uz", "vi", "wo", "xh", "yo", "zh", "zh", "zu"
 }  # fmt: skip
 
+pontoon_languages = { "aa", "aat", "ab", "abb", "abq", "ace", "ach", "ady", "af", "ajg", "ak",
```

Comment:
Since we're hardcoding pontoon lang codes in two places, it's worth moving those to a separate file, maybe somewhere in `pipeline/data/pontoon.py`.